alacritty/src/display.rs

```diff
@@ -407,8 +443,8 @@ impl Display {
             padding_y = dynamic_padding(padding_y, self.size_info.height, cell_height);
         }
 
-        self.size_info.padding_x = padding_x.floor() as f32;
-        self.size_info.padding_y = padding_y.floor() as f32;
+        self.size_info.padding_x = padding_x as f32;
+        self.size_info.padding_y = padding_y as f32;
```

Comment:
Why did you change this? Why aren't things floored anymore?

Reply:
flooring now happens in the function `dynamic_padding`. It's sort of a change I made for myself. It made it easier for me to understand what the values of padding are at each point by enclosing all arithmetics in the function. If you don't like this I'll not be against changing it back.

Reply:
That's a change in functionality though and I don't see any reason for that? We shouldn't change the behavior here unless there's a problem with the old one, but I think the old behavior rounds correctly.

Reply:
Thought the changes in rounding and casting shouldn't cause a problem at our level of precision, but yeah I agree we should avoid unnecessary changes. Changing this back.

Reply:
Every pixel counts, so the only thing that doesn't matter is the floating point values, but rounding incorrectly can create annoying off-by-ones. Most of the rounding is done the way it's done for a very good reason in Alacritty, especially when related to fonts and metrics.

Reply:
It's now changed back.